### PR TITLE
Update links and spec boilerplate to reflect the repository move into WICG.

### PR DESCRIPTION
**Pull request by jeremyroman**

### CODE REVIEW

**Review by domfarolino — APPROVED** [review]
Other Dominic but this popped up in my inbox and LGTM

File: fetch.md [hunk 1]
```diff
@@ -238,8 +238,8 @@ proxy server or virtual private network, if available)
 [1]: https://html.spec.whatwg.org/multipage/webappapis.html#concept-environment-top-level-origin
 [2]: https://fetch.spec.whatwg.org/#network-partition-key
 [3]: https://fetch.spec.whatwg.org/#concept-request-cache-mode
-[4]: https://htmlpreview.github.io/?https://raw.githubusercontent.com/jeremyroman/alternate-loading-modes/spec/index.html#prerendering-browsing-context
+[4]: https://htmlpreview.github.io/?https://raw.githubusercontent.com/WICG/nav-speculation/spec/index.html#prerendering-browsing-context
```
Comment (domenic) [inline fetch.md:241]: This link seems subpar and maybe broken; should we fix it while here?

**Review by domenic — APPROVED** [review]
LGTM as a strict improvement, although I suspect there may have been some broken links due to the split of the specs that we could fix at the same time.

File: fetch.md (hunk 1 above)
Comment (jeremyroman) [reply fetch.md:241]:
```suggestion
[4]: https://wicg.github.io/nav-speculation/prerendering.html#prerendering-browsing-context
```

**Comment by jeremyroman** [comment]
Fixed that one and a couple more.

**Pull request merged**